salesforce/modules/bulk/client.bal

```diff
@@ -91,11 +91,11 @@ public isolated client class Client {
     # + extIdFieldName - Field name of the external ID incase of an Upsert operation
     # + return - returns job object or error
     @display {label: "Create Job"}
-    isolated remote function createJob(@display {label: "Operation"} Operation operation, 
-                                       @display {label: "SObject Name"} string sobj,
-                                       @display {label: "Content Type"} JobType contentType, 
-                                       @display {label: "External ID Field Name"} string extIdFieldName = "") 
-                                       returns @tainted @display {label: "Bulk Job"} error|BulkJob {
+    isolated remote function createJob(@display {label: "Operation"} Operation operation,
+                                        @display {label: "SObject Name"} string sobj,
+                                        @display {label: "Content Type"} JobType contentType,
+                                        @display {label: "External ID Field Name"} string extIdFieldName = "")
```

Comment:
Shall we remove the extra spaces here.

Reply:
This was the result from ballerina formatter. Fixing it manually would take a lot of time